quint/package.json

```diff
@@ -83,6 +83,7 @@
     "apalache-integration": "txm apalache-tests.md",
     "apalache-dist": "txm apalache-dist-tests.md",
     "rust-integration": "txm rust-backend-tests.md",
+    "rust-local-integration": "txm rust-backend-local-tests.md",
     "all-integration": "npm run integration && npm run apalache-integration && npm run rust-integration",
```

Comment:
We need to change this `all-integration`, which is executed on release, to use `rust-local-integration` instead of `rust-integration`.